lib/recurly/apple-pay.js

```diff
@@ -238,7 +262,17 @@ class ApplePay extends Emitter {
   onPaymentAuthorized (event) {
     debug('Payment authorization received', event);
 
-    this.recurly.request('post', '/apple_pay/token', event.payment.token, (err, token) => {
+    // TODO: can we be sure an apple pay form will always have the number field?
+    // var selector = this.recurlyconfig.fields.number.selector;
+    var selector = '[data-recurly=first_name]'
+    var form = dom.findNodeInParents(window.document.querySelector(selector), 'form');
+    var data = normalize(fields, form);
+    var inputs = data.values || {};
```

Comment:
yeap, I was thinking about this too. It also fell to getting something out for Karen to test while deciding how best to get this in (I left the TODO comment as I knew it would draw your attention and you would probably have something to say on the subject). I'll use your idea of passing it in the options to the ApplePay class.